app/packages/core/src/components/ColorModal/JSONViewer.tsx

```diff
@@ -78,10 +79,21 @@ const JSONViewer: React.FC = () => {
         ? data?.showSkeletons
         : colorScheme?.showSkeletons
     );
+    const validatedLabelTags = {
```

Comment:
I'm seeing type errors here. I see you added a `validateLabelTags` function. Could that be leveraged here?